azure_monitor/setup.cfg

```diff
@@ -27,8 +27,8 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api >= 0.1a.0
-    opentelemetry-sdk >= 0.1a.0
+    opentelemetry-api >= 0.2a.0
```

Comment:
Latest is 0.3.dev0

Reply:
The package dependencies depend on `opentelemetry-api` and `opentelemetry-sdk` versions that are RELEASED, not on dev.
https://pypi.org/project/opentelemetry-sdk/
https://pypi.org/project/opentelemetry-api/